packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx

```diff
@@ -73,7 +73,10 @@ export const BookEventForm = ({
 
   // Cloudflare Turnstile Captcha
   const shouldRenderCaptcha =
-    renderCaptcha && CLOUDFLARE_SITE_ID && CLOUDFLARE_USE_TURNSTILE_IN_BOOKER === "1";
+    !process.env.NEXT_PUBLIC_IS_E2E &&
```

Comment:
Found this bug while testing it. In E2e mode the Turnstile component renders null but shouldRenderCaptcha=true was disabling the confirm button.

cc @sean-brydon

Reply:
My bad! Yeah this works